services/blockchain-indexer/shared/indexer/indexStatus.js

```diff
@@ -46,7 +46,7 @@ const getIndexStats = async () => {
 		const currentChainHeight = await getCurrentHeight();
 		const genesisHeight = await getGenesisHeight();
 		const numBlocksIndexed = await blocksDB.count();
-		const [lastIndexedBlock] = await blocksDB.find({ sort: 'height:desc', limit: 1 }, ['height']);
+		const [lastIndexedBlock = {}] = await blocksDB.find({ sort: 'height:desc', limit: 1 }, ['height']);
```

Comment:
Practically, we don't need this change but I think it's better to leave this change for now. This would ensure reporting index status even if the coordinator service is not running/started.

Reply:
Then, please add a TODO to revert the change once the microservice dependency feature is in place.